README.md

```diff
@@ -1,100 +1,28 @@
---optimizer=model_zoo/optimizers/opt_adagrad.prototext \
---num_epochs=5
-```
-Note: `srun -n12 build/gnu.catalyst.llnl.gov/install/bin/lbann` assumes you are running on the LLNL catalyst platform;
-  if running on some other platform, and/or have installed lbann in a different directory, you
-  will need to adjust this command.
-
-  This should produce roughly the following final results on Catalyst:
-```
---------------------------------------------------------------------------------
-[4] Epoch : stats formated [tr/v/te] iter/epoch = [844/94/157]
-            global MB = [  64/  64/  64] global last MB = [  48  /  48  /  16  ]
-             local MB = [  64/  64/  64]  local last MB = [  48+0/  48+0/  16+0]
---------------------------------------------------------------------------------
-Model 0 training epoch 4 objective function : 0.0471567
-Model 0 training epoch 4 categorical accuracy : 99.6241%
-Model 0 training epoch 4 run time : 7.64182s
-Model 0 training epoch 4 mini-batch time statistics : 0.00901458s mean, 0.0212693s max, 0.0078979s min, 0.000458463s stdev
-Model 0 validation objective function : 0.0670221
-Model 0 validation categorical accuracy : 98.9%
-Model 0 validation run time : 0.25341s
-Model 0 validation mini-batch time statistics : 0.00269454s mean, 0.00285273s max, 0.0020936s min, 6.65695e-05s stdev
-Model 0 test objective function : 0.0600125
-Model 0 test categorical accuracy : 99.02%
-Model 0 test run time : 0.421912s
-Model 0 test mini-batch time statistics : 0.00268631s mean, 0.00278771s max, 0.00131827s min, 0.00011085s stdev
-``` 
-  Note: LBANN performance will vary on a machine to machine basis. Results will also vary, but should not do so significantly. 
+A few options for building LBANN are documented [here](docs/BuildingLBANN.md#top).
 
-## Running other models
-There are various prototext models under the lbann/model_zoo/models/ directory: alexnet, autoencoder_mnist, lenet_mnist, etc. Each of these directories should have a script called *runme.py*. Run this script with no command line parameters for complete usage. Basically, these scripts generate command lines similar to the one above (in the *Verifying LBANN on LC* section). The scripts take two required arguments: --nodes=`<int>` and --tasks=`<int>`. The "tasks" option is used to specify the number of tasks per node, hence, the total number of tasks (cores) is: nodes\*tasks. The generated command lines are designed to be executed using *srun* on LC systems, so you may need to modify, e.g, substitute mpirun, depending on your specific system.
 
-Note: some directories contain multiple models, e.g, as of this writing, the autoencoder_cifar10 directory contains both *model_autoencoder_cifar10.prototext* and *model_conv_autoencoder_cifar10.prototext*. In these cases there may be multiple python scripts, e.g, *runme_conv.py*.
+## Running LBANN
+Methods for running LBANN are documented
+[here](docs/RunningLBANN.md#top).
```

Comment:
I know this file doesn't exist. Working on it.

Reply:
You should still include a brief "here's how to run the Hello, World of LBANN" here, with a pointer to more.